### ISSUE TITLE
Definition of semi-automated evaluation

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#385]
Refer to Definition of semi-automated evaluation in Sept-2025 draft of WCAG3.
"Semi-automated evaluation allows machines to guide humans to areas that need inspection. The emerging field of testing conducted via machine learning is not included in this definition".
Comment: Consider another form of semi-automated evaluation: This is where automation works like an expert system  that presents sequential prompts and guides testers to determine presence / absence of violation. I do not think it is covered by "guide humans to areas that need inspection".
e.g. checking for sufficiency / appropriateness of text equivalent for an image, checking for visual focus, checking for focus order, checking for correctness of heading markup, checking for row-column header relationship association with data cells in a complex table etc.
Thanks
Sailesh
